LDAR_Sim/src/file_processing/output_processing/output_utils.py

```diff
@@ -83,6 +84,29 @@ class TIMESERIES_COL_ACCESSORS:
     METH_DAILY_SURVEY_TIME = "{method} Survey Time (Minutes)"
 
 
+def percent_difference(a: float, b: float) -> float:
+    return abs(a - b) / ((a + b) / 2) * 100
+
+
+def relative_difference(aquired_value: float, true_value: float) -> float:
```

Comment:
Same comment as above

Reply:
Will add in handling for the case the reference value is 0